services/databaseService/databaseService.go

```diff
@@ -19,6 +20,12 @@ type userEntry struct {
 var db *sql.DB
 
 func init() {
+
+	if os.Getenv("MYSQL_CONNECTION_STRING") == "" {
+		log.Println("MYSQL_CONNECTION_STRING unassigned. Signing up will not work!")
```

Comment:
Maybe we should only say "Environment variable MYSQL_CONNECTION_STRING is not assigned." -- we might use the database for more than just signing up.